### PR TITLE
add normal CDF to math functions

### PR DESCRIPTION
**Pull request by danfrankj**
PR adds a normal CDF for convenience next to the existing inverse normal CDF function. 

Reference: https://en.wikipedia.org/wiki/Normal_distribution#Cumulative_distribution_function

I have accepted the CLA

### CODE REVIEW

**Comment by findepi** [comment]
@danfrankj could you please add a test?

**Comment by danfrankj** [comment]
@findepi test added :)

**Comment by electrum** [comment]
Please add documentation (in presto-docs directory) for the new function.

**Comment by danfrankj** [comment]
@electrum documentation added

**Comment by danfrankj** [comment]
@electrum @findepi docs and test added ... the CI failure looks like it's unrelated. Anything else you need from me here?

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 1]
```diff
@@ -67,6 +67,12 @@ Mathematical Functions
     a real value and the standard deviation must be a real and positive value.
     The probability p must lie on the interval (0, 1).
 
+.. function:: normal_cdf(mean, sd, v) -> double
+
+    Compute the Normal cdf with given mean and standard deviation (sd):  P(N < v; mean, sd).
```
Comment (findepi) [inline presto-docs/src/main/sphinx/functions/math.rst]: I think `cdf` abbrev should be expanded (same in `inverse_normal_cdf`'s docs). 
@electrum would you agree?

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java [hunk 2]
```diff
@@ -1321,4 +1321,14 @@ public void testInverseNormalCdf()
         assertInvalidFunction("inverse_normal_cdf(4, 48, 1)", "p must be 0 > p > 1");
         assertInvalidFunction("inverse_normal_cdf(4, 0, 0.4)", "sd must > 0");
     }
+
+    @Test
+    public void testNormalCdf()
+            throws Exception
+    {
+        assertFunction("normal_cdf(0, 1, 1.96)", DOUBLE, 0.9750021048517796);
```
Comment (nezihyigitbasi) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: It would be useful to add tests for nan/infinity.

Comment (danfrankj) [reply presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: @nezihyigitbasi I added a few tests for if v = +/- infinity, but not sure how presto handles nans / infinity. What's the convention here? nans produce nans? nans produce errors ... ?

Comment (nezihyigitbasi) [reply presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: Many (most?) Presto functions don't fail with nan input, instead they return a nan value. The return value depends on the function though. In the general case an arbitrary function can also return a non-nan value for a nan input (e.g., java.lang.Double::doubleToLongBits()) .

Comment (danfrankj) [reply presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: @nezihyigitbasi I added several tests relating to nans and infinity, PTAL

**Comment by danfrankj** [comment]
@nezihyigitbasi @electrum @findepi any other changes needed here?

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**